### ISSUE TITLE
Print Datree version before beginning run

### ISSUE THREAD

**Issue by HariSekhon** [datreeio/action-datree#7]
This would make it easier to report issues to datree main repo and fill in the version number in the issue template, as well as to see the version in the link to the github actions build when provided in the ticket.

**Comment by eyarz** [comment]
I agree, but I think it will make more sense to open it as a new issue on Datree's [main repo](https://github.com/datreeio/datree/issues/new?assignees=&labels=enhancement&template=feature_request.md).
But I wonder if it should print on each run or only after installation...?

**Comment by HariSekhon** [comment]
It needs to print version on each run for CI purposes.

This will make it easier for you when you are sent links to output/logs or CI runs in GitHub Actions or some other public hosted provider as you'll be able to see the version in the log.

The reason I've raised this here is that for local datree runs you can just put the version command in the bug report, but for CI links or logs it's better to have it always available.

Also, when a user is using this action-datree, they can't modify it to run the datree version command in their CI pipeline, which is why this needs to be done here.

**Comment by eyarz** [comment]
yep, that makes sense.
@hadar-co can we also print the helm plugin version if someone is using this option with the action?

**Comment by hadar-co** [comment]
Fixed:)

**Comment by HariSekhon** [comment]
Confirmed, this now shows the version before the run, thanks.
